.changelog/2462.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
+resource/mongodbatlas_organization: Fixes organization creation
```

Comment:
Typically, we'd say: Fixes a bug in resource creation where previously <something wasn't right>. Consider adding this bit.